### PR TITLE
Cleanup Mail

### PR DESCRIPTION
**Pull request by sergeyklay**

### CODE REVIEW

File: app/library/Mail.php [hunk 1]
```diff
@@ -32,8 +31,6 @@ public function getTemplate($name, $params)
 		return $this->view->getRender('emailTemplates', $name, $parameters, function($view){
 			$view->setRenderLevel(View::LEVEL_LAYOUT);
 		});
-
-		return $view->getContent();
```
Comment (sergeyklay) [inline app/library/Mail.php:36]: Not needed

**Comment by sergeyklay** [comment]
@duythien

**Comment by sergeyklay** [comment]
Any news?

**Comment by duythien** [comment]
Lol I never see this pull request :)

**Pull request merged**